xskillscore/core/probabilistic.py

```diff
@@ -342,6 +342,78 @@ def threshold_brier_score(
         return res.mean(dim, keep_attrs=keep_attrs)
 
 
+def rps(
+    observations,
+    forecasts,
+    bins,
```

Comment:
Or bin_edges? Optional or required

Reply:
I think this should be `bin_edges` for consistency with the other methods. Also, having this as an optional input defaulting to `np.linspace(0, 1 + 1e-8, 6)` makes sense to me, but I don't feel strongly about this.